plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/status/StatusUtil.java

```diff
@@ -74,6 +74,14 @@ public static MultiStatus multi(Object origin, String message, Throwable error)
     return new MultiStatus(getBundleId(origin), 0, message, error);
   }
 
+  public static MultiStatus multi(Object origin, String message, IStatus[] statuses) {
+    MultiStatus multiStatus = multi(origin, message);
+    for (IStatus status : statuses) {
```

Comment:
Do you think it's worth filtering out OK statuses?  See `filter()` further below.  Otherwise they are reported in the UI as `OK`.

Reply:
Makes sense. I've added a check (`!isOK()`) in `FlexWarStagingDelegate` and `StandardStagingDelegate` to propagate return the multi-status only when it is not OK.

For this util method, I don't think we want to filter OK statuses. It's a general method to construct a multi-status from the given array of statues and not for error handing.